src/schemas/json/sublime-syntax.json

```
@@ -0,0 +1,251 @@
+          "items": {
+            "type": "string"
+          },
+          "minItems": 1
+        },
+        {
+          "$ref": "#/definitions/patterns"
+        }
+      ]
+    },
+    "meta-pattern": {
+      "description": "Meta patterns must be listed first in the context, before any match or include patterns.\nhttps://www.sublimetext.com/docs/syntax.html#meta-patterns",
+      "type": "object",
+      "properties": {
+        "meta_scope": {
+          "description": "This assigns the given scope to all text within this context, including the patterns that push the context onto the stack and pop it off.",
+          "type": "string"
+        },
+        "meta_content_scope": {
+          "description": "This assigns the given scope to all text within this context, including the patterns that push the context onto the stack and pop it off. But does not apply to the text that triggers the context (e.g., in the above string example, the content scope would not get applied to the quote characters).",
+          "type": "string"
+        },
+        "meta_include_prototype": {
+          "description": "Used to stop the current context from automatically including the prototype context.",
+          "type": "boolean"
+        },
+        "clear_scopes": {
+          "description": "This setting allows removing scope names from the current stack. It can be an integer, or the value true to remove all scope names. It is applied before meta_scope and meta_content_scope. This is typically only used when one syntax is embedding another.",
+          "type": "boolean"
+        },
+        "meta_prepend": {
+          "description": "A boolean, controlling context name conflict resolution during inheritance. If this is specified, the rules in this context will be inserted before any existing rules from a context with the same name in an ancestor syntax definition.",
+          "type": "boolean"
+        },
+        "meta_append": {
+          "description": "A boolean, controlling context name conflict resolution during inheritance. If this is specified, the rules in this context will be inserted after to any existing rules from a context with the same name in an ancestor syntax definition.",
+          "type": "boolean"
+        }
+      }
+    },
+    "match-pattern": {
+      "description": "The following keys control behavior that is exclusive, and only one can be specified per match pattern: push pop set embed branch fail\nhttps://www.sublimetext.com/docs/syntax.html#match-patterns",
+      "type": "object",
+      "properties": {
+        "match": {
+          "description": "The regex used to match against the text. YAML allows many strings to be written without quotes, which can help make the regex clearer, but it’s important to understand when you need to quote the regex. If your regex includes the characters #, :, -, {, [ or > then you likely need to quote it. Regexes are only ever run against a single line of text at a time.",
```

Using a proper apostrophe may fix the build in CI:
```suggestion
          "description": "The regex used to match against the text. YAML allows many strings to be written without quotes, which can help make the regex clearer, but it's important to understand when you need to quote the regex. If your regex includes the characters #, :, -, {, [ or > then you likely need to quote it. Regexes are only ever run against a single line of text at a time.",
```